packages/firebase/index.d.ts

```diff
@@ -808,6 +829,29 @@ declare namespace firebase.firestore {
      */
     enablePersistence(): Promise<void>;
 
+    // TODO(multitab): Uncomment when multi-tab is released publicly.
+    // /**
+    //  * Attempts to enable persistent storage, if possible.
+    //  *
+    //  * Must be called before any other methods (other than settings()).
+    //  *
+    //  * If this fails, enablePersistence() will reject the promise it returns.
+    //  * Note that even after this failure, the firestore instance will remain
+    //  * usable, however offline persistence will be disabled.
+    //  *
+    //  * There are several reasons why this can fail, which can be identified by
+    //  * the `code` on the error.
+    //  *
+    //  *   * failed-precondition: The app is already open in another browser tab.
```

Comment:
Technically this should probably have some mention of experimentalTabSynchronization...  but since it's experimental maybe we don't care to give it too much verbiage in this method description...